target/linux/kirkwood/image/Makefile

```diff
@@ -57,6 +57,16 @@ define Device/cloudengines_pogoplugv4
 endef
 TARGET_DEVICES += cloudengines_pogoplugv4
 
+define Device/checkpoint_l50
+  DEVICE_VENDOR := CheckPoint
+  DEVICE_MODEL := L-50
+  DEVICE_DTS := kirkwood-l50
+  DEVICE_PACKAGES := kmod-gpio-button-hotplug kmod-rtc-s35390a kmod-ath9k \
+                    wpad-basic kmod-usb-ledtrig-usbport
+  IMAGE/factory.bin := 
```

Comment:
What's that supposed to do?
Do you actually want to achieve
IMAGES := sysupgrade.bin
instead?

Reply:
Exactly. Thank You.